### PR TITLE
[MRG] DOC: SpaceNet

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Still need to check if anything is missing. Otherwise reviewable.

Thanks for reviewing.

### CODE REVIEW

File: nilearn/decoding/space_net.py [hunk 1]
```diff
@@ -580,22 +602,39 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
...
     `cv_` : list of pairs of lists
-         List of the (n_folds,) folds. For the corresponding fold,
-         each pair is composed of two lists of indices,
-         one for the train samples and one for the test samples.
+        Each pair is the list of indices for the train and test samples
+        for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_alphas, n_folds) or
-                   (n_l1_ratios, n_alphas, n_folds)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
```
Comment (KamalakerDadi) [inline nilearn/decoding/space_net.py]: What is the right way to say about -infs in cv_scores_ as ?
I would like to add a note here about -inf values

File: nilearn/decoding/space_net.py [hunk 2]
```diff
@@ -1021,34 +1061,74 @@ class SpaceNetClassifier(BaseSpaceNet):
...
+    `coef_` : ndarray, shape
+        (1, n_features) for 2 class classification problems (i.e n_classes = 2)
+        (n_classes, n_features) for n_classes > 2
         Coefficient of the features in the decision function.
 
+    `coef_img_` : nifti image
+        Masked model coefficients
+
+    `mask_` : ndarray 3D
+        An array contains masked values.
```
Comment (bthirion) [inline nilearn/decoding/space_net.py]: values of the mask image

File: nilearn/decoding/space_net.py [hunk 3]
```diff
@@ -1208,29 +1288,61 @@ class SpaceNetRegressor(BaseSpaceNet):
...
 
-    `coef_` : array, shape = [n_classes-1, n_features]
+    `coef_` : ndarray, shape (n_features,)
         Coefficient of the features in the decision function.
 
+    `coef_img_` : nifti image
+        Masked model coefficients
+
+    `mask_` : ndarray 3D
+        An array contains masked values
```
Comment (bthirion) [inline nilearn/decoding/space_net.py]: same comment here.

**Comment by bthirion** [comment]
Thx for taking care of this. Looks mostly good to me.

**Comment by bthirion** [comment]
LGTM.

**Comment by bthirion** [comment]
LGTM.

**Comment by GaelVaroquaux** [comment]
This is clearly an improvement over what we have. Merging!

**Pull request merged**